modules/stanford_course_views/stanford_course_views.info

```diff
@@ -9,7 +9,7 @@ dependencies[] = stanford_courses
 dependencies[] = stanford_image
 dependencies[] = views
 dependencies[] = views_bulk_operations
-dependencies[] = views_filters_selective
+dependencies[] = views_selective_filters
```

Comment:
This might be wrong. Not sure what the module is called on dl. Should check.

Reply:
```
drush pmi views_filters_selective
 Extension        :  views_filters_selective                                                                           
 Project          :  views_selective_filters                                                                           
 Type             :  module                                                                                            
 Title            :  Views Selective Exposed Filters                                                                   
 Description      :  Restrict exposed filter values to those present in the result set.                                
 Version          :  7.x-1.5                                                                                           
 Date             :  2016-01-01                                                                                        
 Package          :  Views                                                                                             
 Core             :  7.x                                                                                               
 PHP              :  5.2.4                                                                                             
 Status           :  enabled                                                                                           
 Path             :  sites/default/modules/contrib/views_selective_filters                                             
 Schema version   :  module has no schema                                                                              
 Files            :  views_handler_filter_selective.inc                                                                
 Requires         :  views (>=7.x-3.11), ctools                                                                        
 Required by      :  stanford_course_views, stanford_courses_person_reference_views, stanford_sites_jumpstart_academic 
 Permissions      :  none                                                                                              
 Configure        :  None   
```
The project name is views_selective_filters. The module name is views_filters_selective.

Reply:
Danke.

Reply:
So this should be `views_filters_selective` in the .info file, ja?

Reply:
Jah, .info files need it to be the module name as it lies in the file system. Composer needs it to be the name of the Drupal project as it shows in the url.

Reply:
So this change should be reverted...

Reply:
It has....

Reply:
That's what I get for commenting on an outdated commit 🤦‍♂️